hack/generate-manifests.sh

```diff
@@ -28,7 +30,25 @@ for template in $templates; do
 	$infile > $file
 done
 
+go build -o ${PROJECT_ROOT}/tools/csv-generator/csv-generator ${PROJECT_ROOT}/tools/csv-generator/csv-generator.go
+file="${DEPLOY_DIR}/vm-import-operator/${VERSION}/vm-import-operator.${CSV_VERSION}.clusterserviceversion.yaml"
+rendered=$( \
+	${PROJECT_ROOT}/tools/csv-generator/csv-generator \
+	--csv-version=${CSV_VERSION} \
+	--replaces-csv-version=${CSV_VERSION_REPLACES} \
+	--namespace=${KUBEVIRT_NAMESPACE} \
+	--operator-version=${CONTAINER_TAG} \
+	--operator-image="${CONTAINER_PREFIX}/${OPERATOR_IMAGE}" \
+	--image-pull-policy=${IMAGE_PULL_POLICY} \
+)
+if [[ ! -z "$rendered" ]]; then
+	echo -e "$rendered" > $file
+fi
+
+(cd ${PROJECT_ROOT}/tools/csv-generator && go clean)
+
 # TODO: Will be useful once examples are generated per version
+exit 0
```

Comment:
Why is here this exit? Maybe some testing leftover?

Reply:
the condition below never met. I'll remove the 'exit' and the dead code beneath